Sources/Controllers/2.0/Settings/NavigationSettings/OAVehicleParametersViewController.mm

```diff
@@ -140,28 +140,27 @@ - (void) setupView
             }
         }
     }
-    // TODO: add default speed functionality when it's ready
-//    if (self.appMode != OAApplicationMode.PUBLIC_TRANSPORT && self.appMode.parent != OAApplicationMode.PUBLIC_TRANSPORT)
-//    {
-//        if (self.appMode != OAApplicationMode.AIRCRAFT && self.appMode.parent != OAApplicationMode.AIRCRAFT)
-//            [defaultSpeedArr addObject:@{
-//                @"type" : kCellTypeIconText,
-//                @"title" : OALocalizedString(@"default_speed"),
-//                @"minSpeed" : [NSNumber numberWithDouble:router->getMinSpeed()],
-//                @"defaultSpeed" : [NSNumber numberWithDouble:router->getDefaultSpeed()],
-//                @"maxSpeed" : [NSNumber numberWithDouble:router->getMaxSpeed()],
-//                @"icon" : @"ic_action_speed",
-//                @"name" : @"defaultSpeed",
-//            }];
-//        else
-//            [defaultSpeedArr addObject:@{
-//                @"type" : kCellTypeIconText,
-//                @"title" : OALocalizedString(@"default_speed"),
-//                @"defaultSpeedOnly" : @YES,
-//                @"icon" : @"ic_action_speed",
-//                @"name" : @"defaultSpeed",
-//            }];
-//    }
+    if (self.appMode != OAApplicationMode.PUBLIC_TRANSPORT && self.appMode.parent != OAApplicationMode.PUBLIC_TRANSPORT)
+    {
+        if (self.appMode != OAApplicationMode.AIRCRAFT && self.appMode.parent != OAApplicationMode.AIRCRAFT)
+            [defaultSpeedArr addObject:@{
+                @"type" : kCellTypeIconText,
+                @"title" : OALocalizedString(@"default_speed"),
+                @"minSpeed" : [NSNumber numberWithDouble:router->getMinSpeed()],
```

Comment:
This and other places use incorrect methods. The only methods you should use are: ```[self.appMode getDefault/Min/MaxSpeed] and [self.appMode setDefault/Min/MaxSpeed]```